server/functions.lua

```diff
@@ -234,39 +286,62 @@ end
 
 -- Callback Functions --
 
--- Client Callback
+---Trigger Client Callback
+---@param name string
+---@param source any
+---@param cb function
+---@param ... any
 function QBCore.Functions.TriggerClientCallback(name, source, cb, ...)
     QBCore.ClientCallbacks[name] = cb
     TriggerClientEvent('QBCore:Client:TriggerClientCallback', source, name, ...)
 end
 
--- Server Callback
+---Create Server Callback
+---@param name string
+---@param cb function
 function QBCore.Functions.CreateCallback(name, cb)
     QBCore.ServerCallbacks[name] = cb
 end
 
+---Trigger Serv er Callback
+---@param name string
+---@param source any
+---@param cb function
+---@param ... any
 function QBCore.Functions.TriggerCallback(name, source, cb, ...)
     if not QBCore.ServerCallbacks[name] then return end
     QBCore.ServerCallbacks[name](source, cb, ...)
 end
 
 -- Items
 
+---Create a usable item
+---@param item string
+---@param data function
```

Comment:
Is `function` correct here? Or would this be a table?

Reply:
The usage for this function is to add a callback function as the second parameter. I did consider changing the argument to `cb` like it used to be.

It was changed from `cb` to `data` here: https://github.com/qbcore-framework/qb-core/pull/813